ansible_runner/__main__.py

```diff
@@ -217,6 +217,15 @@
                      "(status and stdout still included for other events)"
             ),
         ),
+        (
+            ("--no-store-env",),
+            dict(
+                default="True",
```

Comment:
The default value here is `True` for "not" storing the env, but your code in `__init__.py` says `kwargs.get('store_env', True)`.

I'm personally not a fan of CLI options that start with `--no-`... they are confusing. I think I'd prefer to change this to `--store-env`.

Reply:
One of my goals in writing this was to maintain backwards compatibility which is `kwargs.get('store_env', True)`. Initially I was going to do an inputted parameter like: `--store-env <True|False>` but that ended up seeming like extra typing since it defaults to `True` you would only ever be typing `--store-env False`. If I just go with `--store-env` the default (no parameter) would break this backwards compatibility. Because of these two factors I ended up going with `--no-` flag. Line 223 should be removed; it really does nothing and was from the initial go at the <True|False> form of the parameter. If we don't want to go with `--no` I wouldn't be against renaming the parameter to something like `--skip-env` and propagating that through the code (i.e. `kwargs.get('skip_env', False)`. Or we could go back to the text boolean parameter option of `--store-env <True|False>`. Or, if we are ok with changing the default behavior, we could also do `--store-env` to go back to the previous default behavior. Let me know what you think.

Reply:
Ok, what you've said here makes sense. I was mostly confused about `default="True"`. Remove that and we are good.